### ISSUE TITLE
fasta: does not report runtime error when final field initialize twice (initializer and constructor)

### ISSUE THREAD

**Issue by mraleph** [dart-lang/sdk#28566]
```
Language/Classes/Constructors/Generative_Constructors/execution_of_an_initializer_t02.dart
Language/Classes/Constructors/Generative_Constructors/initializing_formals_execution_t02.dart
Language/Variables/final_t01/01
Language/Variables/final_t02/01
``` 

/cc #28263

**Comment by mraleph** [comment]
Still the issue with fasta.

**Comment by peter-ahe-google** [comment]
I have a local fix for this.

**Comment by peter-ahe-google** [comment]
Fixed in 470127ae46ca783f7d625010c028073464cdb179.
